app/views/admin/dashboards/show.html.erb

```diff
@@ -1,79 +1,90 @@
-<h4 class="govuk-heading-l">Admin dashboard</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <h1 class="govuk-heading-l">Dashboard</h1>
+  </div>
 
-<nav>
-  <%= link_to "Upload building list", new_admin_bulk_import_path, class: "govuk-button" %>
-  <%= link_to "Add a building record", new_admin_building_path, class: "govuk-button" %>
-</nav>
+  <div class="govuk-grid-column-one-half dashboard-actions">
+    <%= link_to "Upload building list", new_admin_bulk_import_path, class: "govuk-button" %>
+    <%= link_to "Add a building record", new_admin_building_path, class: "govuk-button" %>
+  </div>
+</div>
+
+<% if flash[:alert] %>
```

Comment:
a separate partial would be nice but also not urgent

Reply:
Agree - have done this.